nav2_system_tests/models/turtlebot3_waffle_depth_camera/model.sdf

```diff
@@ -335,7 +335,7 @@
 
       <sensor name="intel_realsense_r200_depth" type="depth">
         <always_on>1</always_on>
-        <update_rate>1</update_rate>
+        <update_rate>15</update_rate>
```

Comment:
Given this is for a systems test, you _sure_ you want to make that 15hz? That's going to be pretty CPU intensive.

Reply:
1 fps is not enough and useless. 30 fps causes some slow down, but I have been testing with 15 fps. It works as long as you don't try to visualize the point cloud and voxel layer markers.

Reply:
+ I changed the robot model name. the system test uses the default tb3 model for now.